### PR TITLE
[JENKINS-72148] Add missing fr properties for win32errors

### PR DESCRIPTION
**Pull request by jgreffe**
See [JENKINS-72148](https://issues.jenkins.io/browse/JENKINS-72148).
Add and translate all the missing *_fr.properties for win32errors.

### Proposed changelog entries

- JENKINS-72148, add missing fr properties for win32errors

### Proposed upgrade guidelines

N/A

```[tasklist]
### Submitter checklist
- [x] The Jira issue, if it exists, is well-described.
- [ ] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)). Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [ ] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.
```

### Desired reviewers

@mention

Before the changes are marked as `ready-for-merge`:

```[tasklist]
### Maintainer checklist
- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).
```

### CODE REVIEW

**Comment by lemeurherve** [comment]
Review in progress (that file is so long 😭)

**Comment by jgreffe** [comment]
> Review in progress (that file is so long 😭)

Indeed. Note the "disquette" I didn't write for a long time 🤣

File: core/src/main/resources/hudson/win32errors_fr.properties [hunk 1]
```diff
@@ -0,0 +1,2074 @@
...
+error1011= \
+La clé du registre de configuration ne peut pas être ouverte
+error1012= \
+La clé du registre de configuration ne peut pas être lue
+error1013= \
+La clé du registre de configuration ne peut pas être écrite
+error1014= \
+Un des fichiers de la base de donnée du registre a dû être récupéré par l''utilisation d''un log ou d''une copie alternative. La récupération est un succèe
+error1015= \
+Le registre est corrompu. La structure de l''un des fichiers contenant les données du registre est corrompue, ou l''image mémoire système du fichier est corrompue, ou le fichier n''a pû être récupéré parce que la copie alternative ou le log était absent ou corrompu
```
Comment (lemeurherve) [inline core/src/main/resources/hudson/win32errors_fr.properties:2058]:
```suggestion
Le registre est corrompu. La structure d''un des fichiers contenant les données du registre est corrompue, ou l''image mémoire système du fichier est corrompue, ou le fichier n''a pû être récupéré parce que la copie alternative ou le log était absent ou corrompu
```

File: core/src/main/resources/hudson/win32errors_fr.properties [hunk 2]
```diff
@@ -0,0 +1,2074 @@
...
+error1016= \
+Une opération I/O initiée par le registre a échoué de façon irrécupérable. Le registre n''a pas pû lire ou écrire ou vider un des fichiers qui contient l''image système du registre
```
Comment (lemeurherve) [inline core/src/main/resources/hudson/win32errors_fr.properties:2060]:
```suggestion
Une opération I/O initiée par le registre a échoué de façon irrécupérable. Le registre n''a pas pû lire, écrire ou vider un des fichiers contenant l''image système du registre
```

File: core/src/main/resources/hudson/win32errors_fr.properties [hunk 3]
```diff
@@ -0,0 +1,2074 @@
...
+error1017= \
+Le système a tenté de charger ou de restaurer un fichier dans le registre, mais le fichier spécifié ne possède par le format de fichier de registre
```
Comment (lemeurherve) [inline core/src/main/resources/hudson/win32errors_fr.properties:2062]:
```suggestion
Le système a tenté de charger ou de restaurer un fichier dans le registre, mais le fichier spécifié n''est pas dans un format de fichier de registre
```

File: core/src/main/resources/hudson/win32errors_fr.properties [hunk 4]
```diff
@@ -0,0 +1,2074 @@
...
+error1018= \
+Tentative d''opération illégale sur une clé de registre qui a été indiquée pour suppression
+error1019= \
+Le système n''a pas pû allouer l''espace requis dans un log de registre
```
Comment (lemeurherve) [inline core/src/main/resources/hudson/win32errors_fr.properties:2066]:
```suggestion
Le système n''a pas pû allouer l''espace requis dans un journal de registre
```

File: core/src/main/resources/hudson/win32errors_fr.properties [hunk 5]
```diff
@@ -0,0 +1,2074 @@
...
+error101= \
+Le sémaphore exclusif est la propriété d''un autre processus
+error102= \
+Le sémaphore est déclaré et ne peut être fermé
+error103= \
+Le sémaphore ne peut être déclaré de nouveau
+error104= \
+Impossible de requêter des sémaphores exclusifs lors de moments d''interruption
+error105= \
+Le sémaphore a été libéré"
```
Comment (lemeurherve) [inline core/src/main/resources/hudson/win32errors_fr.properties:237]:
```suggestion
Le sémaphore a été libéré
```

Comment (lemeurherve) [reply core/src/main/resources/hudson/win32errors_fr.properties:237]: Sorry if it has been introduced by one of my previous suggestions.

**Review by lemeurherve — COMMENTED** [review]
A misplaced `"` has been introduced, probably because of me, all good otherwise.

**Review by lemeurherve — APPROVED** [review]
That wasn't an easy one, congrats 👏

**Review by NotMyFault — APPROVED** [review]
Thanks!

**Pull request merged**